src/Control/RequestHandler.php

```diff
@@ -657,7 +658,7 @@ public function getReferer()
     public function redirectBack()
     {
         // Don't cache the redirect back ever
-        HTTP::set_cache_age(0);
+        HTTPCacheControlMiddleware::singleton()->disableCache(true);
```

Comment:
I was previously [advocating](https://github.com/silverstripe/silverstripe-framework/pull/8086#issuecomment-395241798) for moving this logic into the middleware, since this decision can be made by inspecting the response. It's how the Laravel and Symfony HTTP caching middlewares work. The advantage here is that you can tailor behaviour through the middleware with high granularity, e.g. only applying redirect caching for a certain route. It's currently not possible to influence this here in `RequestHandler` unless we add some config flags which make the entire system more complex.